### PR TITLE
MODSOURMAN-779: Add status for Import jobs that are stopped by users.

### PR DESCRIPTION
**Pull request by VRohach**
## Purpose
Added new status "CANCELLED" for job that are stopped by users manually.

## Approach
Extended mechanism for status and ui-status execution

## Learning
https://issues.folio.org/browse/MODSOURMAN-779

### CODE REVIEW

**Comment by VRohach** [comment]
This PR can be merged only after merging: https://github.com/folio-org/data-import-raml-storage/pull/258

**Review by SerhiiNosko — APPROVED** [review]

**Review by RuslanLavrov — APPROVED** [review]

**Review by okolawole-ebsco — APPROVED** [review]

**Review by Aliaksandr-Fedasiuk — APPROVED** [review]

**Pull request merged**
